<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="mask">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Маска файла</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="fileDelete">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалять файл после преобразования</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Директория сохранения</string>
    </property>
   </widget>
   <widget class="QComboBox" name="nameRepeat">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Повторяющиеся файлы</string>
    </property>
   </widget>
   <widget class="QLabel" name="modeLabel">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Режим запуска</string>
    </property>
   </widget>
   <widget class="QComboBox" name="executeMode">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="timer">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="timerLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Таймер</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xorKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>XOR ключ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="executeButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>251</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Запустить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
